### PR TITLE
Add angle brackets around links that contain spaces to increase compatibility

### PR DESCRIPTION
**Pull request by christianselig**
Hey! Love the project, only catch was for my Reddit app I was getting a few users complaining my Markdown renderer (which uses GitHub's renderer) doesn't render some of your links properly, e.g.: https://www.reddit.com/r/apolloapp/comments/s5tsk9/link_markdown_broken_i_think_it_has_to_do_with/

The "issue" (if you'd call it that) is that when you create the URL to PM you with, you include some JSON there that has spaces in it, which breaks GitHub's Markdown renderer. The Reddit website handles this properly somehow, but I thought I'd make a quick PR that wraps the URLs in <angled brackets> which per Commonmark (which Reddit also supports) [allows you to add spaces](https://superuser.com/questions/1170654/how-do-i-add-a-hyperlink-with-spaces-in-it-using-markdown) to URLs for increased compatibility. I updated the tests to reflect this.

tl;dr: shouldn't break anything but should increase compatibility.

### CODE REVIEW

**Comment by barrycarey** [comment]
Hey, thanks for the info.  I had no idea that was a problem. 

Changes look good.  I should be able to test them this weekend just to make sure.

**Comment by christianselig** [comment]
That's awesome to hear! Let me know if there's anything I can do to help :D
